### PR TITLE
Delete fetch

### PR DESCRIPTION
**Pull request by Drewbrad4**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -67,19 +66,84 @@ export default class App extends React.Component {
       })
   }
 
-  createNewBook = (form) => {
-    console.log(form);
+  createNewBook = (newBook) => {
+    console.log(newBook)
```
Comment (sturdynut) [inline app/javascript/components/App.js:70]: 🔥  Don't want to leave console logs in our final code, so if this isn't needed, should be removed.

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -67,19 +66,84 @@ export default class App extends React.Component {
...
+        console.log("delete errors:", errors)
+      })
   }
 
   findItem = (arr, id) => {
     return arr.find(item => item.id === Number(id))
   }
 
+  findRentedBooks = (arr, id) => {
+    let rentedBooks = arr.filter(book => book.rentals.length > 0)
```
Comment (sturdynut) [inline app/javascript/components/App.js:135]: I really love how you made 3 methods for doing this.  Turned out really nice.  I also like how you split this up into 2 calls.  It makes it easier to understand what is happening, step-by-step.  Just as an example of another approach, you could simplify this down to this:

```
let rentedBooks = arr.filter(book => book.rentals.length > 0 && book.rentals[0].user_id === id);
return rentedBooks;
```

You can make it even shorter by using an implicit return like so:

```
findRentedBooks = (arr, id) => arr.filter(book => book.rentals.length > 0 && book.rentals[0].user_id === id)
```

What you have is great, i'm just trying to show you some other ways of doing this.  👏

File: app/javascript/components/App.js [hunk 3]
```diff
@@ -198,24 +268,43 @@ export default class App extends React.Component {
...
               let id = props.match.params.id
               let book = this.state.books.find(book => book.id === parseInt(id))
-              return (
-                <LendEdit
-                  updateBook={this.updateBook}
-                  current_user={current_user}
-                  book={book}
-                />
-              )
+              console.log(book)
```
Comment (sturdynut) [inline app/javascript/components/App.js:292]: 🔥  - Do you need this console log in here?

File: app/javascript/components/Components/Header.js [hunk 4]
```diff
@@ -5,14 +5,10 @@ const Header = ({ sign_out_route }) => (
 
     <div id="header">
 
-        <h3 className="title">PageLender</h3>
+        <a href="/"><h3 className="title">PageLender</h3></a>
```
Comment (sturdynut) [inline app/javascript/components/Components/Header.js:8]: Not sure if you were intentionally using an `<a />` here, but you may want to consider using the `react-router-dom` [Link](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/Link.md) instead.

The main difference is that the anchor tag will reload your entire application where as the react-router-dom `<Link />` will not, so your page won't flicker and you'll keep your state.

**Review by sturdynut — APPROVED** [review]
🥳  Such Awesomeness 🤩

Great work!  Left you some things to consider and read more about.

File: app/javascript/components/App.js (hunk 3 above)
Comment (Drewbrad4) [reply app/javascript/components/App.js:292]: Nope, thanks!

File: app/javascript/components/App.js (hunk 2 above)
Comment (Drewbrad4) [reply app/javascript/components/App.js:135]: Oh duh! Seeing the rework, that makes total sense. That reminded me that I can actually just make all the helpers one-liners so I did. Thanks!

File: app/javascript/components/Components/Header.js (hunk 4 above)
Comment (Drewbrad4) [reply app/javascript/components/Components/Header.js:9]: Yes, thanks for the reminder. Switched those.

**Pull request merged**